tests/cli/test_compose_tar_kickstart.sh

```diff
@@ -37,6 +37,14 @@ description = "tar image test"
 version = "0.0.1"
 modules = []
 
+[[groups]]
+name = "anaconda-tools"
+version = "*"
```

Comment:
groups have no version, but it also shouldn't really matter, it will just be ignored.